splink/cluster_studio.py

```diff
@@ -252,6 +252,7 @@ def render_splink_cluster_studio_html(
 
     files = {
         "embed": "files/external_js/vega-embed@6.20.2",
+        "stdlib": "files/external_js/stdlib.js@5.8.3",
```

Comment:
I was unable to find a way of using stdlib without inlineing it, hence the inclusion